assets/js/util/insufficient-permissions-error-description.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * getInsufficientPermissionsErrorDescription function.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x, sprintf } from '@wordpress/i18n';
+
+/**
+ * Gets a description for an insufficient permissions error.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} error Original error message.
+ * @param {string} moduleName Module name.
+ * @param {string} ownerName Module owner name.
+ * @return {string} Error description.
+ */
+export function getInsufficientPermissionsErrorDescription( error, moduleName, ownerName ) {
+	let message = '';
+	let userInfo = '';
+
+	if ( _x( 'Analytics', 'Service name', 'google-site-kit' ) === moduleName ) {
+		if ( error.match( /account/i ) ) {
+			message = __( `Your Google account does not have sufficient permissions for this Analytics account, so you won't be able to see stats from it on the Site Kit dashboard.`, 'google-site-kit' );
+		} else if ( error.match( /property/i ) ) {
+			message = __( `Your Google account does not have sufficient permissions for this Analytics property, so you won't be able to see stats from it on the Site Kit dashboard.`, 'google-site-kit' );
+		} else if ( error.match( /view/i ) ) {
+			message = __( `Your Google account does not have sufficient permissions for this Analytics view, so you won't be able to see stats from it on the Site Kit dashboard.`, 'google-site-kit' );
+		}
+	} else if ( _x( 'Search Console', 'Service name', 'google-site-kit' ) === moduleName ) {
+		message = __( `Your Google account does not have sufficient permissions for this Search Console property, so you won't be able to see stats from it on the Site Kit dashboard.`, 'google-site-kit' );
```

Comment:
Using a placeholder makes the translatable string harder to translate. These four messages are good as is because they are easy to understand and translate. Plus, there is no need for a placeholder because module names for these messages are constant and won't change.